usbsdmux/usbsdmux.py

```diff
@@ -97,16 +180,131 @@ def mode_DUT(self, wait=True):
         self._pca.output_values(self._DAT_enable | self._PWR_enable | self._select_DUT | self._card_removed)
 
     def mode_host(self, wait=True):
-        """
-        Switches the MicroSD-Card to the Host.
-
-        This Command will issue a disconnect first to make sure the the SD-card
-        has been properly disconnected from both sides and it's supply was off.
-        """
         self.mode_disconnect(wait)
 
-        # the disconnect-command has already switched the card to the host.
-        # Thus we don't need to worry about glitches anymore.
+        # The disconnect-command has already switched the card to the host.
+        # Thus, we don't need to worry about glitches anymore.
 
         # now connect data and power
         self._pca.output_values(self._DAT_enable | self._PWR_enable | self._select_HOST | self._card_inserted)
+
+    def gpio_get(self, gpio):
+        raise NotImplementedError()
+
+    def gpio_set_high(self, gpio):
+        raise NotImplementedError()
+
+    def gpio_set_low(self, gpio):
+        raise NotImplementedError()
```

Comment:
Maybe have the `UsbSdMux` base class raise NotImplementedError for all of its methods and just not implement them in this derived class?

Reply:
With the `` @abc.abstractmethod`` decorator on the base class you are not able to create an object from the derived class when not all of these methods have been overridden.
It's not quite the same, but still close :-)
Feel free to re-open.

Reply:
There is some benefit in having it explicitly as not implemented in the derived class.

Reply:
@jluebbe Than I am not sure how you want to implement this. What `abc` decoration would you suggest?

Reply:
I meant that we can leave it as is.